src/Php72/Php72.php

```diff
@@ -97,16 +97,33 @@ public static function spl_object_id($object)
 
     public static function sapi_windows_vt100_support($stream, $enable = null)
     {
+        if (!is_resource($stream)) {
+            trigger_error('sapi_windows_vt100_support() expects parameter 1 to be resource, '.gettype($stream).' given', E_USER_WARNING);
+            return false;
+        }
+
```

Comment:
Hey thanks for fixing my oversight. `stream_isatty` needs a similar resource check, if you wouldn't mind.

Reply:
Sure, should i include it in this PR?